packages/kmod-5.15-nvidia/kmod-5.15-nvidia.spec

```
@@ -282,8 +283,15 @@ install -m 755 nvidia-cuda-mps-control %{buildroot}%{_cross_bindir}
 install -m 755 nvidia-cuda-mps-server %{buildroot}%{_cross_bindir}
 install -m 755 nvidia-persistenced %{buildroot}%{_cross_bindir}
 install -m 4755 nvidia-modprobe %{buildroot}%{_cross_bindir}
+ln -s ../../../../bin/nvidia-smi %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/nvidia-smi
```

should we make these paths absolute, like

```suggestion
ln -s %{buildroot}%{_cross_bindir}/nvidia-smi %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/nvidia-smi
```
?

---

Actually, I doubled-checked other places where we create symlinks and they use the flag `-r` to create a relative symlink:

```suggestion
ln -rs %{buildroot}%{_cross_bindir}/nvidia-smi %{buildroot}%{_cross_libexecdir}/nvidia/tesla/bin/nvidia-smi
```

See [here] for reference.

[here]: https://github.com/search?q=repo%3Abottlerocket-os%2Fbottlerocket-core-kit+path%3A*.spec+ln+-rs&type=code

(I'm sorry @sky1122!)

---

Thank you for the suggestions! changed.